tests/wallet.rs

```diff
@@ -165,22 +181,22 @@ fn send_inscribed_sat() {
 
 #[test]
 fn send_on_mainnnet_refuses_to_work_with_wallet_name_foo() {
-  let rpc_server = test_bitcoincore_rpc::builder().wallet_name("foo").build();
-  let txid = rpc_server.mine_blocks(1)[0].txdata[0].txid();
+  let rpc_server = test_bitcoincore_rpc::builder().build();
 
   CommandBuilder::new(
-    format!("wallet send bc1qzjeg3h996kw24zrg69nge97fw8jc4v7v7yznftzk06j3429t52vse9tkp9 {txid}:0:0"),
```

Comment:
We need to somehow test that send fails.

Reply:
(You can create the wallet with the bitcorerpc Client directly, and then test that send refuses to operate)